README.md

```diff
@@ -58,12 +58,18 @@ Initial Tweet: https://twitter.com/swisscom_csirt/status/1341388348389244934
   | [ir-rescue](https://github.com/diogo-fernan/ir-rescue)                                     | :cloud:                      | :cloud:                                                  | :sunny:                 | :sunny: <br /> Commercial usage needs permission             | :sunny:                                              | :sunny: <br /> (Bash v4+)                                        | :sunny:        | :cloud: <br /> [AVML](https://github.com/microsoft/avml) for memory dump                       | :cloud:                                                         | :sunny:                               | :sunny:                                                 | 
   | [CyLR](https://github.com/orlikoski/CyLR)                                         | :sunny:                      | :sunny:                                                   | :cloud:                 | :sunny:              | :sunny:  | :partly_sunny: <br /> .NET code and recompilation                                                                               | :sunny:        | :sunny: <br />.NET Binary                                                 | :cloud:  | :partly_sunny: <br />[Open Letter to the users](https://docs.google.com/document/d/1L6CBvFd7d1Qf4IxSJSdkKMTdbBuWzSzUM3u_h5ZCegY/edit?usp=sharing)                                                             | :sunny:                                                 | 
   | [artifactcollector](https://github.com/forensicanalysis/artifactcollector)                          | :cloud:                      | :sunny: <br />[Forensics Artifact Repository](https://github.com/ForensicArtifacts/artifacts)                                                   | :sunny:                 | :sunny:              | :sunny:                                              | :partly_sunny: <br /> Prepare artifacts in YAML and Go compilation                                           | :sunny:        | :sunny:          | :cloud:                                                                                                               | :sunny:                                                       | :partly_sunny: <br />ArtefactStore                                                | 
-  
+  | [DFIR_Linux_Collector](https://github.com/xophidia/DFIR_Linux_Collector) | :cloud: | :sunny: | :sunny: | :sunny: | :sunny: | :sunny: | :sunny: <br />(Bash) | :cloud: | ? | :sunny: | :sunny: (text, json, raw) |
```

Comment:
output parsing: :cloud: because it doesn't offer anything related to artifact parsing.

Reply:
easy extensible: I would make :partly_sunny: or :cloud: (which you prefer) because it needs a recompilation after adding a new artifact or tool and that requires an analyst is ready to do that.